robonomics/protocol/src/subxt/datalog.rs

```diff
@@ -43,3 +45,24 @@ where
     );
     Ok(xt_hash.into())
 }
+
+/// Read datalog records from remote Robonomics node.
+pub async fn fetch(robot_account: AccountId, remote: String) -> Result<Vec<RingBufferItem>> {
```

Comment:
Let’s return primitive types, in that case code that use it don’t need to known about pallet internals and it’s changes.